modules/serverless-install-web-console.adoc

```diff
@@ -1,12 +1,17 @@
 // Module included in the following assemblies:
 //
-// * /serverless/installing_serverless/installing-openshift-serverless.adoc
+// * /serverless/admin_guide/install-serverless-operator.adoc
 
 [id="serverless-install-web-console_{context}"]
 = Installing the {ServerlessOperatorName}
 
 This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
 
+[IMPORTANT]
+====
+Before upgrading to the latest Serverless release, you must remove the community Knative Eventing Operator if you have previously installed it. Having the Knative Eventing Operator installed will prevent you from being able to install the latest version of Knative Eventing using the {ServerlessOperatorName}.
```

Comment:
I'll go through contortions to avoid future tense.  I think you _could_ do something like "If the Knative Eventing Operator is installed, <xyz> prevents you from installing the latest version..."